doc/sphinx-guides/source/admin/big-data-administration.rst

```diff
@@ -0,0 +1,312 @@
+Scaling Dataverse with Data Size
+================================
+
+This guide is intended to help administrators configure Dataverse appropriately to handle the amount of data their installation needs to manage.
```

Comment:
```suggestion
This guide is intended to help administrators configure Dataverse appropriately to handle larger amounts of data.
```

Also, we could say "section" since this is already inside a guide, the Admin Guide.